CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.0)
+
+include("cmake/HunterGate.cmake")
+HunterGate(
+  URL "https://github.com/ruslo/hunter/archive/v0.12.44.tar.gz"
+  SHA1 "54963db4cd6e2a8780643a3cc48bdbbefedec6cc"
+)
+
+hunter_add_package(ZLIB)
```

Comment:
> I tested it and it worked for me

That is strange. Can you try clean version? Like
```
> rm -rf _builds _HUNTER
> cmake -H. -B_builds -DHUNTER_ROOT="`pwd`/_HUNTER" -DHUNTER_STATUS_DEBUG=ON -DCMAKE_VERBOSE_MAKEFILE=ON
```

Do you have hunter cloned to the `HUNTER_ROOT` or you just use default settings?